src/cpp/session/modules/rmarkdown/SessionRMarkdown.cpp

```diff
@@ -1301,19 +1301,27 @@ Error getRmdTemplate(const json::JsonRpcRequest& request,
 
    json::Object jsonResult;
 
-   // locate the template skeleton on disk (if it doesn't exist we'll just
-   // return an empty string)
-   FilePath skeletonPath = FilePath(path).completePath("skeleton/skeleton.Rmd");
+   // locate the template skeleton on disk
+   // (return empty string if none found)
    std::string templateContent;
-   if (skeletonPath.exists())
+   for (auto&& suffix : { "skeleton/skeleton.Rmd", "skeleton/skeleton.rmd" })
    {
-      error = readStringFromFile(skeletonPath, &templateContent, string_utils::LineEndingPosix);
-      if (error)
-         return error;
+      FilePath skeletonPath = FilePath(path).completePath(suffix);
+      if (skeletonPath.exists())
+      {
+         error = readStringFromFile(skeletonPath, &templateContent, string_utils::LineEndingPosix);
+         if (error)
+         {
+            LOG_ERROR(error);
+            continue;
+         }
+      }
+
+      jsonResult["content"] = templateContent;
+      break;
```

Comment:
I don't think this logic is correct -- it looks like `skeleton/skeleton.rmd` will never be read except under very unusual circumstances (the Rmd version exists but can't be read). Am I misreading this?

Reply:
Sorry, you are correct. I've fixed this up now.